Content.Server/Atmos/EntitySystems/GasTileOverlaySystem.cs

```diff
@@ -215,15 +221,17 @@ private bool UpdateChunkTile(GridAtmosphereComponent gridAtmosphere, GasOverlayC
             }
 
             var changed = false;
+            var temp = tile.Hotspot.Valid ? tile.Hotspot.Temperature : tile.Air?.Temperature ?? Atmospherics.TCMB;
```

Comment:
Should we be looking at the hotspot temperature, or just the gas temperature? It doesn't seem like we should make the whole tile show the heat effect if we just have a little wielder flame.

Reply:
So when I was trying to figure out what "Hotspot" referred to -- I think I came to the conclusion that a tile with a "Hotspot" had the full tile flame effect. I've tested it as well and a welding flame does not cause a tile to have a heat effect.

I think I remember I initially had trouble because the "Hotspot" temperature could be distinct from the temperature of the air mixture in a tile, but at least for this effect the "Hotspot" temp is what is relevant.